ConnectorAutocadCivil/ConnectorAutocadCivil/UI/ConnectorBindingsAutocadCivil.cs

```diff
@@ -1001,18 +1136,42 @@ private void ConvertSendCommit(Base commitObject, ISpeckleConverter converter, S
                 converted["propertySets"] = propertySets;
 #endif
 
-              string containerName = obj is BlockReference ?
-                "Blocks" :
-                Utils.RemoveInvalidDynamicPropChars(layer); // remove invalid chars from layer name
+              string containerName = obj is BlockReference ? "Blocks" : Utils.RemoveInvalidDynamicPropChars(layer); // remove invalid chars from layer name
 
               if (commitObject[$"@{containerName}"] == null)
                 commitObject[$"@{containerName}"] = new List<Base>();
               ((List<Base>)commitObject[$"@{containerName}"]).Add(converted);
 
               // set application id
+              #region backwards compatibility
```

Comment:
@teocomi this is the main change for handling old acad application ids which have already been saved on the obj